pom.xml

```diff
@@ -836,7 +836,7 @@
       <dependency>
         <groupId>com.github.luben</groupId>
         <artifactId>zstd-jni</artifactId>
-        <version>1.5.6-6</version>
+        <version>1.5.6-7</version>
```

Comment:
Is this released without tagging? It seems that the last tag is still `1.5.6-6`.
- https://github.com/luben/zstd-jni/tags

Reply:
Maybe, is this not announced officially yet?

Reply:
Yes, I think it's just that it hasn't been announced officially yet. But its GitHub has already been marked as follows:
https://github.com/luben/zstd-jni
<img width="548" alt="image" src="https://github.com/user-attachments/assets/269432b1-7373-4384-bc1c-b67a2b2f53a6">